### PR TITLE
Add basic templates to be inserted on new perl6 (and related) files

### PR DESCRIPTION
**Pull request by Tyil**

### CODE REVIEW

File: templates/meta [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "perl" : "6.c",
+    "name" : "Nondescript",
+    "version" : "0.0.1",
+    "description" : "Nondescript.",
+    "authors" : [
+        "John Doe"
```
Comment (samcv) [inline templates/meta]: Maybe put "Author Name" instead?

File: templates/meta [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "perl" : "6.c",
+    "name" : "Nondescript",
```
Comment (samcv) [inline templates/meta]: "name" : "ModuleName"

File: templates/meta [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "version" : "0.0.1",
+    "description" : "Nondescript.",
```
Comment (samcv) [inline templates/meta]: "description" : "Module Description"

File: templates/meta [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    "authors" : [
+        "John Doe"
+    ],
+    "license" : "GPL-3.0",
```
Comment (samcv) [inline templates/meta]: "license" : "Artistic-2.0" ?

Since that is what the vast majority of Perl 6 modules are.

**Review by samcv — CHANGES_REQUESTED** [review]
Few changes that wolud be good.

File: templates/meta (hunk 4 above)
Comment (Tyil) [reply templates/meta]: But GPL is freedom, and I want to promote freedom :(

File: templates/meta (hunk 1 above)
Comment (Tyil) [reply templates/meta]: I used "Jonh Doe" because that's the standard way to refer to an anonymous name, but I can change this.

**Comment by Tyil** [comment]
This should go to vim-perl/perl6 if anything.

**Pull request closed without merging**